feather/server/src/lib.rs

```diff
@@ -98,6 +98,9 @@ impl Server {
     pub fn accept_new_players(&mut self) -> Vec<ClientId> {
         let mut clients = Vec::new();
         for player in self.new_players.clone().try_iter() {
+            if let Some(cl) = self.clients.iter().find(|x| x.uuid() == player.uuid) {
+                cl.disconnect("Logged in from another location!");
```

Comment:
Looks like we need to `return` here, or else the new client gets added anyway.

Reply:
We are not removing the new client but instead the old one. Thus returning would probably just break everything.